dlrover/python/common/constants.py

```diff
@@ -325,7 +326,7 @@ class RendezvousName(object):
 
 
 class NodeErrorMessage(object):
-    NETWORKER_ERROR = "Network is breakdown"
+    NODE_CHECK_ERROR = "Node check has failure"
```

Comment:
"Node check has failure" -> "Node check not passed"

Reply:
"Node check failed"